documentation/public/user-guide/query-language.md

```diff
@@ -0,0 +1,109 @@
+# The Tree Query Language
+
+## Query Syntax
```

Comment:
I think the new docs are using the old "name" terminology instead of "labels" (changed in #857).
I suggest standardizing on it, since users will be reading/using `NodeLabel` in the APIs.

Reply:
Done in https://github.com/NomicFoundation/slang/pull/928/commits/dc3eab1a23788c85037df9b54a3da9f8503f26f5